StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/NavigationBar/LinkSheetNavigationBar.swift

```diff
@@ -97,8 +97,8 @@ class LinkSheetNavigationBar: SheetNavigationBar {
     }
 
     override func createBackButton() -> UIButton {
-        let image = Image.icon_x_standalone.makeImage(template: true)
-        let resizedImage = Image.icon_chevron_left_standalone.makeImage(template: true).resized(to: CGSize(width: LinkUI.navigationBarButtonContentSize, height: LinkUI.navigationBarButtonContentSize))
```

Comment:
The usage of `icon_x_standalone` was incorrect before, but never showed up since `resizedImage` took precedence.